.gitattributes

```diff
@@ -0,0 +1,9 @@
+/.github export-ignore
+/.vscode export-ignore
+/tests export-ignore
+/examples export-ignore
+.coveralls.yml export-ignore
```

Comment:
A couple minor asks:

1. Since you're using the prefixing `/` for the first set of these, would you mind just using it for all of them?
2. Can you alphabetize the list?
3. Can you add a short comment to the top of the file indicating that the purpose of this list is to make sure that these files are excluded from the final Composer package.
4. Can you squash down your commits?

Reply:
@brandur-stripe 
Done